mos.yml

```diff
@@ -541,16 +543,43 @@ conds:
         STOCK_FW_MODEL: '"SHRGBW2-color"'
         # We don't use SSL, HomeKit uses its own crypto. This saves ~120K.
         MG_ENABLE_SSL: 0
+        MIN_COLOR_TEMPERATURE_VALUE: 50
```

Comment:
where do these come from? will there be variations between device types? i don't see these defined elsewhere.